about.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+  <title>The role of technology in tutoring</title>
+</head>
+
+<body class="body">
+  <section class="landing-page">
+    <header class="header-container">
+      <a href="index.html" class="my-logo"><img src="./images/logo.png" alt="This is my Portfolio Logo"></a>
+      <nav class="navigation">
+        <a href="" class="my-menu" title="menu bar"></a>
+        <ul class="desktop-nav">
+          <li class="nav-men">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-men">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+        </ul>
+        <div id="mobilenav" class="mobileNavigation">
+          <a href="javascript:void(0)" class="killnav" onclick="menukill()">&times;</a>
+          <a href="index.html" class="close-menu">Home</a>
+          <a href="about.html" class="close-menu">About</a>
+        </div>
+        <span class="navonmo" id="menu-toggle"><img src="./images/Union (2).png" alt="menu nav bar" style="width: 40px; height: 40px";></span>
+      </nav>
+    </header>
+    <div class="landing-page-div">
+      <h2 class="homepage-h2">"Hello! Sharing world"</h2>
+      <h1 class="homepage-h1">Revolutionizing Tutoring Through Technology Summit 2023  <img class="lineonabt" src="images/line.png" alt="line"></h1>
```

Comment:
> - [ ] I kindly suggest you use CSS background-image or a border line to implement the line since the image is not part of the content rather than using an `img` tag. This is for aesthetic reasons. You can [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.👍 Also ensure the size is the same as the provided design.☺